doc/source/suse-ecp/ose-localhost.rst

```diff
@@ -125,27 +125,6 @@ configuration file loading ARA plugins:
 For more details on ARA's web interface, please read
 https://ara.readthedocs.io/en/stable/webserver.html .
 
-Enable Mitogen (optional)
```

Comment:
this is the part for ecp environment, target audient is devs. this option can be useful here? @Itxaka @evrardjp  your thoughts?

Reply:
This is optional everywhere.
It's largely making the ansible deploys faster.

The documentation (before rewrite) was not ccp specific, and only ONE part (configure your hosts for socok8s) was partially on ccp (with an option to bring your own). So I think the refactor could have made that more unclear.

I will review this in the whole context of the docs later, if you like.

Reply:
For me it was hanging on some random tasks so I had to disable it fully to complete a deploy.

If we recommend this in the docs we *need* to support it, so I would say leave it out as it would make troubleshooting more difficult.

Reply:
Sounds good to me.